test.py

```diff
@@ -0,0 +1,65 @@
+import os
+import pytest
+import subprocess
+import testinfra
+
+@pytest.fixture(scope='session')
+def host(request):
+    subprocess.check_call(['docker', 'build', '-t', 'radicale', '.'])
+    docker_id = subprocess.check_output(['docker', 'run', '-d', 'radicale']).decode().strip()
+    
+    yield testinfra.get_host("docker://" + docker_id)
+    
+    # teardown
+    subprocess.check_call(['docker', 'rm', '-f', docker_id])
+
+def test_system(host):
```

Comment:
I would skip this test

Reply:
Yeah, added it because I could, but it doesn't offer much value

Reply:
Ah, actually one reason it does add value is in the build.sh. It'd be easy to have one of the ARCH builds on a different version of alpine by mistake. This would ensure that doesn't happen